gitlab-pipeline/stage/post.yml

```diff
@@ -43,3 +43,40 @@ coveralls:finish-build:
   script:
     - 'curl -k ${COVERALLS_WEBHOOK_URL}?repo_token=${MENDER_COVERALLS_TOKEN} -d "payload[build_num]=$(git rev-parse HEAD)&payload[status]=done"'
 
+.trigger_template:
+  stage: .post
+  rules:
+    # - if: '$INTEGRATION_REV =~ /^[0-9]+\.[0-9]+\.[0-9]+(-build\d+)?$/'
+    - when: always
+    # the following is to prevent an endless loop of qa pipelines caused by downstream pipelines
+    - if: '$CI_PIPELINE_SOURCE == "pipeline"'
+      when: never
+
+trigger:mender-convert:
+  extends: .trigger_template
+  variables:
+    MENDER_ARTIFACT_VERSION: $MENDER_ARTIFACT_REV
+    MENDER_CLIENT_VERSION: $MENDER_REV
+    MENDER_ADDON_CONNECT_VERSION: $MENDER_CONNECT_REV
+    MENDER_ADDON_CONFIGURE_VERSION: $DEVICECONFIG_REV
+  trigger:
+    project: Northern.tech/Mender/mender-convert
+    branch: pr_356
+    strategy: depend
+
+trigger:mender-binary-delta:
+  extends: .trigger_template
+  variables:
+    MENDER_VERSION: $MENDER_REV
+    MENDER_ARTIFACT_VERSION: $MENDER_ARTIFACT_REV
+  trigger:
+    project: Northern.tech/Mender/mender-binary-delta
+    branch: pr_92
+    strategy: depend
```

Comment:
I am sorry this was wrong in the JIRA description, but I think now that we neither need this one here.

`mender-binary-delta` is released completely independent and not part of the regular "release test plan". Therefore we don't need to link the pipelines. @kacf Can you please confirm? I think it was in the slide that you presented but when I think about it now it doesn't make sense to me...

Reply:
👌 removed

Reply:
For the release, it is something we want, because the tests use both mender-artifact and the Yocto image to test with. But if it makes sense on *every* build? I don't think so, we probably aren't gonna look at it.

Reply:
And it's also important to get all build artifacts out of one pipeline.